xml/deploy_cephadm.xml

```diff
@@ -701,131 +701,287 @@ o- time_server ................................................ [enabled]
+       appear, you can configure a <emphasis>registry cache</emphasis>.
       </para>
-     </step>
-    </procedure>
-    <tip>
-     <title>Registry cache</title>
+     </tip>
+    </sect4>
+    <sect4 xml:id="deploy-cephadm-configure-imagepath">
+     <title>Configuring the path to container images</title>
+     <tip>
+      <title>Configuring HTTPS proxy</title>
+      <para>
+       If you need to use a proxy to communicate with the container registry
+       server, perform the following configuration steps on all cluster nodes:
+      </para>
+      <procedure>
+       <step>
+        <para>
+         Copy the configuration file for containers:
+        </para>
+<screen>&prompt.sudo;cp /usr/share/containers/containers.conf /etc/containers/containers.conf</screen>
+       </step>
+       <step>
+        <para>
+         Edit the newly-copied file and add the <option>http_proxy</option>
+         setting to its <literal>[engine]</literal> section, for example:
+        </para>
+<screen>&prompt.user;cat /etc/containers/containers.conf
+ [engine]
+ http_proxy=proxy.example.com
+ [...]
+ </screen>
+       </step>
+      </procedure>
+     </tip>
      <para>
-      To avoid re-syncing the local registry when new updated containers
-      appear, you can configure a <emphasis>registry cache</emphasis>.
-<!-- 2021-02-09 tbazant: still looking for better solution
-      Find
-      more information about containers and air-gapped scenarios in
-      <link
-       xlink:href="https://documentation.suse.com/sbp/all/single-html/SBP-Private-Registry/#_deployment_in_an_air_gapped_environment"/>.
-      -->
+      &cephadm; needs to know a valid URI path to container images. Verify the
+      default setting by executing
      </para>
-    </tip>
-    <para>
-     Cloud Native Application Development and Delivery methods require a
-     registry and a CI/CD (Continuous Integration/Delivery) instance for the
-     development and production of container images. You can use a private
-     registry in this instance.
-<!-- 2021-02-09 tbazant: still looking for better solution
-     For more information, see
-     <link
-       xlink:href="https://documentation.suse.com/sbp/all/single-html/SBP-Private-Registry/"/>.
-     -->
-    </para>
+<screen>&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_image_path ls</screen>
+     <para>
+      If you do not need an alternative path or to create a local container
+      registry, you can leave the default or set it when the setting has no
+      default path:
+     </para>
+<screen>&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_image_path set registry.suse.com/ses/7/ceph/ceph</screen>
+     <para>
+      If your deployment requires a specific path, for example, a path to a
+      local registry, configure it as follows:
+     </para>
+<screen>&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_image_path set <replaceable>LOCAL_REGISTRY_PATH</replaceable></screen>
```

Comment:
Is the "path" here a URL? If it is, does it need the `https://` prefix or something else? I think we should specify this clearly.

Reply:
Nope. It's the path to the image on the container registry, which is (irritatingly) kinda like a URL, in that it's got host, possibly port, and path components, but there's no protocol component (i.e. no https:// or other prefix)